src/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,21 @@
+import '@testing-library/jest-dom';
+import { render, screen, cleanup } from '@testing-library/react';
+import Calculator from '../components/calculator';
+
+describe('Calculator tests', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  test('Calculator page render', () => {
+    render(<Calculator />);
+    const acBtn = screen.getByText('AC');
+    expect(acBtn).toBeInTheDocument();
+  });
+
+  test('Calculator calculations', () => {
```

Comment:
- Please add test scenarios for: `+/-`, `.`, `=` operators. It's mandatory that you test all functionalities in this module.

Reply:
I have these tests in the operator test